src/lib/es2017.intl.d.ts

```diff
@@ -1,5 +1,5 @@
 declare namespace Intl {
-    type DateTimeFormatPartTypes = "day" | "dayPeriod" | "era" | "hour" | "literal" | "minute" | "month" | "second" | "timeZoneName" | "weekday" | "year";
+    type DateTimeFormatPartTypes = "day" | "dayPeriod" | "era" | "fractionalSecond" | "hour" | "literal" | "minute" | "month" | "second" | "timeZoneName" | "weekday" | "year";

```

Comment:
suggestion from @weswigham:

```suggestion
    interface DateTimeFormatPartTypesRegistry {
        day: any
        dayPeriod: any
        // etc ...
    }
    type DateTimeFormatPartTypes = keyof DateTimeFormatPartTypesRegistry;
```

Then in es2021.intl.dts, add

```ts
    interface DateTimeFormatPartTypesRegistry {
       fractionalSecond: any
    }
```